tutorials/code_submitter.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: How to Submit Code
+---
+
+# How to Submit Code
+
+For virtual aspects of the competition, such as the [Virtual League][virtual-league], you will submit your code using the [code-submitter][code-submitter].
+Your code will then be run alongside other teams' code in the [simulator][simulator].
+
+To prepare your code for upload, you should create a zip archive containing all the code you want to submit.
+As with running the code in the simulator locally or on your robot, this must include a file named _exactly_ **`robot.py`**, placed in the root of the archive (not within a subfolder).
+You can also provide other files as well and use them from your `robot.py` file, however the code-submitter will always look for a `robot.py` file as that is what the simulator will run.
+
+Once you have prepared the archive containing your code:
+
+1. Visit the [Code Submitter][code-submitter]
```

Comment:
Don't think all these numbers should be 1.

Reply:
I can see why you'd ask, however this is a deliberate markdown trick to avoid redundant diff noise if a new item needs adding in the middle of the list. It'll still render correctly. The numeric markers just mean "this is a numbered list" -- the individual numbers used are ignored by the rendering logic in either case.

Reply:
Ah, ok. I'm not familiar with all markdown tricks.